roles/ora-host/defaults/main.yml

```diff
@@ -72,6 +72,62 @@ oracle_required_rpms:
   - xorg-x11-utils
   - xorg-x11-xauth
 
+oracle_required_rpms_oel8:
```

Comment:
Nit:  These requirements likely also apply to RHEL8, so I'd call this "oracle_required_rpms_el8".  If you feel more ambitious, you could use a nested list like oracle_required_rpms["el8"]: https://stackoverflow.com/questions/46854348/accessing-elements-on-a-list-of-lists